css/selectors/nesting.json

```
@@ -5,15 +5,15 @@
         "__compat": {
           "description": "Nesting selector (<code>&</code>)",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Nesting_selectors",
-          "spec_url": "https://drafts.csswg.org/css-nesting/#nesting-selector",
+          "spec_url": "https://drafts.csswg.org/css-nesting-1/#nest-selector",
```

We try to use version-less CSS specs. Is there a reason for this change?

```suggestion
          "spec_url": "https://drafts.csswg.org/css-nesting/#nest-selector",
```

---

I changed this in content as it would not load the spec without the _-1_. I have now learnt that this is not needed in BCD.